project.clj

```diff
@@ -13,7 +13,8 @@
                  [secretary "1.2.3"]]
 
   :plugins [[lein-environ "1.0.2"]
-            [lein-asset-minifier "0.2.8"]]
+            [lein-asset-minifier "0.2.8"]
+            [lein-cljfmt "0.3.0"]]
```

Comment:
I copied it from another repo as opposed to checking the latest version, hehe. Does lein ancient complain? I can update it.